frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -92,6 +100,32 @@ pub mod pallet {
 			VaultId = Self::VaultId,
 		>;
 
+		/// Currency is used for the assets managed by the vaults.
+		type Currency: Transfer<Self::AccountId, Balance = Self::Balance, AssetId = Self::AssetId>
+			+ Mutate<Self::AccountId, Balance = Self::Balance, AssetId = Self::AssetId>
+			+ MutateHold<Self::AccountId, Balance = Self::Balance, AssetId = Self::AssetId>;
+	
+		type Pablo: Amm<
+			AssetId = Self::AssetId,
+			Balance = Self::Balance,
+			AccountId = Self::AccountId,
+			PoolId = Self::PoolId,
+		>;
+
+		// Type representing the unique ID of a pool.
+		type PoolId: FullCodec
+			+ MaxEncodedLen
+			+ Default
+			+ Debug
+			+ TypeInfo
+			+ Eq
+			+ PartialEq
+			+ Ord
+			+ Copy
+			+ Zero
+			+ One
+			+ SafeArithmetic;
```

Comment:
Is it bad practice to use traits that are not being used yet? Can't this save you from unnecessary mistakes in the future?

Reply:
I don't know in case of Substrate (/cc @kevin-composable), but usually unnecessary dependencies do not result in anything good later (dependency hell for example).

> Can't this save you from unnecessary mistakes in the future?

What do you mean?

Reply:
@kevin-composable 
@belousm 

Bump!

Reply:
Resolved by ad5e3c755504c6505053ff3953bec98dbe39b63b.